package.json

```diff
@@ -4,8 +4,8 @@
   "description": "[![atlp-rwanda](https://circleci.com/gh/atlp-rwanda/Team-furebo-E-commerce-bn.svg?style=svg)](https://app.circleci.com/pipelines/github/atlp-rwanda)",
   "main": "index.js",
   "scripts": {
-    "test": "nyc --reporter=lcov --reporter=text mocha src/tests/*.test.js --timeout 60000 --require @babel/register --exit && codeclimate-test-reporter < coverage/lcov.info",
-    "coverage": "nyc report --reporter=text-lcov | tee coverage.lcov | coveralls && ./node_modules/.bin/codeclimate-test-reporter < coverage/lcov.info",
```

Comment:
Why have you removed these lines for running codeclimate when the tests run and the coverage on code climate ?